backend/apps/monitoring/celery_monitor.py

```
@@ -0,0 +1,265 @@
+
+        broker_url = os.getenv("CELERY_BROKER_URL", "redis://localhost:6379/0")
+        if broker_url.startswith("redis"):
+            r = redis.Redis.from_url(broker_url, socket_timeout=1.0)
+            for q in queues_to_check:
+                try:
+                    length = r.llen(q)
+                    if length > 0 or q in ["default", "celery"]:
+                        stats_data["queues"][q] = length
+                        stats_data["total_queue_depth"] += length
+                except Exception:
+                    pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)